### PR TITLE
fix: pt_br language bug in permissions

### LINKED ISSUE #2062: [BUG] Some texts already translated to PT-BR a long time ago still appear in English on Alby UI
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

Some texts already translated to PT-BR a long time ago still appear in English on Alby UI

### Screenshots [optional]

The texts:

1. Read your public key.
2. Decrypt data.
3. Sign message with your key.
4. Encrypt data.

was already translated to PT-BR a long time ago but still appear in English for me.

Captura de tela de 2023-02-04 10-58-33

### Steps To Reproduce

1. Change Alby to PT-BR language
2. Go to a site with Nostr support like nostrich.com
3. Click on Alby edit permission settings.

### Expected behavior

See the texts in Portuguese Brazilian

### Alby information

Alby: 1.25.1

### PR DESCRIPTION
**Pull request by manavdesai27**
### Describe the changes you have made in this PR

Fixed language error in permissions menu while using `pt_br`
### Link this PR to an issue [optional]

Fixes #2062 

### Type of change

- `fix`: Bug fix (non-breaking change which fixes an issue)

### Screenshots of the changes [optional]

image
### How has this been tested?

_Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration_

### Checklist

- [x] My code follows the style guidelines of this project and performed a self-review of my own code
- [x] New and existing tests pass locally with my changes
- [x] I checked if I need to make corresponding changes to the documentation (and made those changes if needed)

### CODE REVIEW

**Comment by manavdesai27** [comment]
@im-adithya Could you review? Thanks!

**Comment by manavdesai27** [comment]
@escapedcat Could you review please?

File: src/i18n/i18nConfig.ts [hunk 1]
```diff
@@ -38,6 +38,7 @@ export const resources = {
     translation: pt_BR.translation,
     common: pt_BR.common,
     components: pt_BR.components,
+    permissions: pt_BR.permissions,
```
Comment (escapedcat) [inline src/i18n/i18nConfig.ts:41]: Ah, nice catch! Would you mind extending all entries which are missing `permissions`? Thanks!

Comment (manavdesai27) [reply src/i18n/i18nConfig.ts:41]: Sure! I will extend this to other languages as well!

Comment (manavdesai27) [reply src/i18n/i18nConfig.ts:43]: @escapedcat Done!

File: src/i18n/locales/zh_Hans/translation.json [hunk 2]
```diff
@@ -674,5 +674,13 @@
       "used": "已使用",
       "payments": "付款"
     }
+  },
+  "permissions": {
+    "nostr": {
+      "getpublickey": "阅读您的公钥。",
```
Comment (escapedcat) [inline src/i18n/locales/zh_Hans/translation.json:680]: No need to add the translations. Better let native speakers do these later on. In this case it's "formal you" but the rest is using the non-formal one for example.
你 VS 您

Comment (manavdesai27) [reply src/i18n/locales/zh_Hans/translation.json:680]: Well okay, but won't the code break if I don't add the permissions in these files?

Comment (escapedcat) [reply src/i18n/locales/zh_Hans/translation.json:680]: No, [Weblate](https://hosted.weblate.org/git/getalby-lightning-browser-extension/getalby-lightning-browser-extension/) will add empty translations which will be filled by people translating there language. Fallback is always English.

Comment (escapedcat) [reply src/i18n/locales/zh_Hans/translation.json:680]: Thanks for thinking about this though!

Comment (manavdesai27) [reply src/i18n/locales/zh_Hans/translation.json:680]: Okay, lemme revert back.

Comment (escapedcat) [reply src/i18n/locales/zh_Hans/translation.json:680]: You can just remove the translations and leave the structure.

Comment (escapedcat) [reply src/i18n/locales/zh_Hans/translation.json]: Arg sorry, leave the structure otherwise the type-check fails. Will merge right after. My fault.

Comment (manavdesai27) [reply src/i18n/locales/zh_Hans/translation.json]: Is this what you want? The code falls back to English as you stated but the IDE shows the squiggles over permissions. And the checks also gives out the same code quality issue. I could let the code have `"permissions": {}` so it doesn't break.

Comment (manavdesai27) [reply src/i18n/locales/zh_Hans/translation.json]: Done!

**Pull request merged**